src/api/status/public/js/build-log/check-for-build.js

```diff
@@ -4,6 +4,9 @@ import buildHeader from './build-header.js';
 import showToast from '../utils/toast.js';
 
 let build;
+let uploaded = false; //a flag to check if build log is added to supabase
+
+const statusUrl = (path) => `http://localhost:1111${path}`;
```

Comment:
How to make `status` know which url to use or how to pass `process.env.STATUS_URL` to this public js

Reply:
Well, you could use one small trick. Since this is client JS, what if you grab the [URL of the window](https://developer.mozilla.org/en-US/docs/Web/API/Location) and use that? I think that should be fine.

Reply:
However, if you follow @manekenpix's idea, you might not need to do this.